pykern/pkcli/__init__.py

```diff
@@ -187,20 +189,42 @@ def _argh_name_mapping_policy():
 
 
 def _commands(cli):
-    """Extracts all public functions from `cli`
+    """Extracts all public functions or methods from `cli`
 
     Args:
-        cli (module): where commands are executed from
+        cli (object): where commands are executed from
 
     Returns:
-        list of function: public functions sorted alphabetically
+        list: cmomands sorted alphabetically
     """
-    res = []
-    for n, t in inspect.getmembers(cli):
-        if _is_command(t, cli):
-            res.append(t)
-    sorted(res, key=lambda f: f.__name__.lower())
-    return res
+
+    def _exclude():
```

Comment:
I think a longer name would be helpful. Maybe `_methods_of_base_classes`. I could guess what __bases__ is but had to look it up. I also don't like names that know how a function will be used.

Reply:
[c2074bc](https://github.com/radiasoft/pykern/pull/500/commits/c2074bc5baefa1061edd6f114b61338a96786c2a)